Formula/zola.rb

```diff
@@ -29,21 +28,28 @@ def install
     ENV["RUSTONIG_SYSTEM_LIBONIG"] = "1"
     system "cargo", "install", "--features", "native-tls", *std_cargo_args
 
-    bash_completion.install "completions/zola.bash"
-    zsh_completion.install "completions/_zola"
-    fish_completion.install "completions/zola.fish"
+    mkdir buildpath/"completions" do
+      system "#{bin}/zola completion bash > zola.bash"
+      bash_completion.install "zola.bash"
+
+      system "#{bin}/zola completion fish > zola.fish"
+      fish_completion.install "zola.fish"
+
+      system "#{bin}/zola completion zsh > _zola"
+      zsh_completion.install "_zola"
```

Comment:
You can simplify this with https://rubydoc.brew.sh/Formula#generate_completions_from_executable-instance_method

Reply:
Oh shucks, I just saw that mentioned the other day in another PR but it apparently left my mind in the interim time. Thanks for the reminder!